src/storage/storageResolver.js

```diff
@@ -0,0 +1,161 @@
+'use strict'
```

Comment:
add overlay, we will first look at it and completing value with the storage resolver